src/web/components/SharingPermission/BulkAddPermissions.tsx

```diff
@@ -12,28 +14,32 @@ import {
   getDefaultDataProviderCheckboxState,
   getDefaultDSPCheckboxState,
   getDefaultPublisherCheckboxState,
+  getFilteredParticipantsByType,
   getRecommendationMessageFromTypeNames,
   getRecommendedTypeFromParticipant,
   hasUncheckedASharedType,
 } from './bulkAddPermissionsHelpers';
+import { ParticipantItemSimple } from './ParticipantItem';
 
 import './BulkAddPermissions.scss';
 
 type BulkAddPermissionsProps = {
   participant: ParticipantDTO | null;
   hasSharingParticipants: boolean;
+  availableParticipants: AvailableParticipantDTO[];
   onBulkAddSharingPermission: (selectedTypes: string[]) => Promise<void>;
   sharedTypes: string[];
 };
 
 export function BulkAddPermissions({
   participant,
   hasSharingParticipants,
+  availableParticipants,
```

Comment:
We are retrieving all sites from the admin service? and based on the selected type to filter on them?

Reply:
that is the plan yes! load all of them in and then just filter by type within this component